query/common/src/main/java/org/codice/ddf/admin/common/services/ServiceCommons.java

```diff
@@ -214,4 +221,16 @@ public void setServiceReader(ServiceReader serviceReader) {
     public void setConfiguratorFactory(ConfiguratorFactory configuratorFactory) {
         this.configuratorFactory = configuratorFactory;
     }
+
+    public static void updateGraphQLSchema(Class clazz) {
+        getEventAdmin(clazz).postEvent(UPDATE_SCHEMA_EVENT);
+    }
+
+    public static EventAdmin getEventAdmin(Class clazz) {
+        return getBundleContext(clazz).getService(getBundleContext(clazz).getServiceReference(EventAdmin.class));
```

Comment:
Any reason we didn't use ServiceReader.getServiceReference(Class) method to get the EventAdmin?